elasticapm/transport/base.py

```diff
@@ -257,7 +263,7 @@ def close(self):
         self._closed = True
         self.queue("close", None)
         if not self._flushed.wait(timeout=self._max_flush_time):
-            raise ValueError("close timed out")
+            logger.error("Closing the transport connection timed out.")
```

Comment:
I wonder if we additionally also should use a shorter timeout than `max_flush_time`. Maybe 2s?

Reply:
Hmmm. I can potentially see the reasoning, but let's assume the user has upped the `api_request_time` because they're on a slow network (or for some other reason). I don't particularly want to lose data due to being impatient at close time.

I think I'm going to leave it, unless you think there's a hole in my reasoning?